src/styl/_variables/_color.styl

```diff
@@ -57,7 +57,7 @@
     --color-fade: $color-background-3-dark
     --color-borders: $color-foreground-3-dark
 
-    --color-links: $color-links
+    --color-links: $color-theme-default-dark
```

Comment:
I don't like it.
It meens it's not settable via colette settings.
I think we should use a `$color-links-dark`